<launch>

  <arg name="robot_name" default="robot1"/>

  <group ns="$(arg robot_name)">

    <!-- load the joint state controller -->
    <rosparam file="$(find abb_irb120_gazebo)/config/joint_state_controller.yaml" command="load" />
    <node name="joint_state_controller_spawner" pkg="controller_manager" type="spawner" 
    args="joint_state_controller"/>

    <!-- load the arm controller -->
    <rosparam command="load" file="$(find abb_irb120_gazebo)/config/irb120_3_58_arm_controller.yaml">
    </rosparam>

    <node name="abb_irb120_controller_spawner" pkg="controller_manager" type="spawner" args="arm_controller"/>

  </group>
</launch>
